test/dns_nsupdate/dns_nsupdate_test.rb

```diff
@@ -1,8 +1,6 @@
 require 'test_helper'
-require 'dns/dns'
-require 'dns_nsupdate/dns_nsupdate_plugin'
+require 'dns_common/dns_common'
 require 'dns_nsupdate/dns_nsupdate_main'
-require 'dns_nsupdate/dns_nsupdate_gss_main'
```

Comment:
I don't see any gss-specific tests in this suite, so this require can be safely dropped.